docs/dev/Build.md

```diff
@@ -196,7 +196,7 @@ sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.
 sudo dnf -y update
 
 # Install required tools and libraries
-sudo dnf install -y autoconf automake boost-devel clang cmake curl gcc gcc-c++ \
+sudo dnf install -y autoconf automake boost169-devel clang cmake curl gcc gcc-c++ \
```

Comment:
why boost169 here? this is for centos/rhel8. please revert back

Reply:
@cxxman because otherwise boost v.166 is install and on centos7 we are using boost169-devel. Why not using the same version?

Reply:
we should use "default" version if it fits for our needs, version 1.66 is ok, on the centos7 there is some earlier version which doesn't fit, that's why we install 1.69